framework-crates/objc2-io-usb-host/translation-config.toml

```diff
@@ -5,9 +5,9 @@ custom-lib-rs = true
 macos = "10.15"
 maccatalyst = "14.0"
 
-# Return type marked as non-null, raises `unknown error result type`.
-class.IOUSBHostControllerInterface.methods."getPortStateMachineForCommand:error:".skipped = true
-class.IOUSBHostControllerInterface.methods."getPortStateMachineForPort:error:".skipped = true
+# Return type nullability is incorrectly declared in the header. It should be `nullable` instead of `nonnull`.
+class.IOUSBHostControllerInterface.methods."getPortStateMachineForCommand:error:".return.nullability = "nullable"
+class.IOUSBHostControllerInterface.methods."getPortStateMachineForPort:error:".return.nullability = "nullable"
```

Comment:
Nit: Could I get you to re-run the translator, and commit the result? You don't have to open a PR to `objc2-generated`, as long as you've pushed it to GH then I can find it when merging.

If it's too bothersome, then I can do it for you?

Reply:
Sure, no problem, it's here https://github.com/carlossless/objc2-generated/commit/b8d328c65b921049a9faacf6b97256aad501417f

Reply:
I meant, you have to link it in the commit in this PR. I think something like `git add generated`

Reply:
Oh, I wasn't aware this could work this way. Is this what you mean?

Reply:
Yup, exactly.